src/main/controllers/features/limited/companyInactiveController.ts

```diff
@@ -3,21 +3,21 @@ import * as config from "../../../config";
 import { selectLang, addLangToUrl, getLocalesService, getLocaleInfo } from "../../../utils/localise";
 import { LIMITED_WHAT_IS_THE_COMPANY_NUMBER, LIMITED_COMPANY_INACTIVE, BASE_URL } from "../../../types/pageURL";
 import { Session } from "@companieshouse/node-session-handler";
-import { COMPANY } from "../../../common/__utils/constants";
+import { COMPANY_DETAILS } from "../../../common/__utils/constants";
 import { Company } from "../../../model/Company";
 
 export const get = async (req: Request, res: Response, next: NextFunction) => {
 
     const session: Session = req.session as any as Session;
-    const company : Company = session?.getExtraData(COMPANY)!;
+    const company : Company = session?.getExtraData(COMPANY_DETAILS)!;
 
     const lang = selectLang(req.query.lang);
     const locales = getLocalesService();
 
     res.render(config.LIMITED_COMPANY_INACTIVE, {
         previousPage: addLangToUrl(BASE_URL + LIMITED_WHAT_IS_THE_COMPANY_NUMBER, lang),
         startPage: addLangToUrl(BASE_URL, lang),
-        title: "Company is not active",
+        title: "You cannot register this company",
```

Comment:
Does this title needs to come from language json file?

Reply:
No